templates/change.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Changes to profile</title>
+</head>
+<body>
+<!---->
+<form>
+<p>Time
+        <input name="time" id= "time" type="checkbox" checked>
+    </p>
+    <p>Twitter
+        <input name="twittertoken" id="twitter" type="checkbox">
+    </p>
+     <p>Calendar
+     <input name="calendar" id="calendar" type="checkbox">
+    </p>
+    <p>Map
+        <input name="maps" id="eta" type="checkbox">
+    </p>
+    <p>Spotify
+        <input name="spotifytoken" id="spotify" type="checkbox">
+    </p>
+    <input type="Submit" value="Submit Change" />
+</form>
+
+<!--Will make js that promts pin and if matches will call delete account-->
+<button onclick="promtPin">Delete</button>
```

Comment:
Is 'promptPin' a placeholder for now? Or an actual method somewhere?

Reply:
Placeholder

Reply:
promt is a command in js that launches an input box figured there should be a way to match it with the pin stored on the backend but wasn't 100%